src/components/RuleCard/index.jsx

```diff
@@ -218,430 +218,433 @@ function RuleCard({ rule, onRuleDelete, readOnly, ...props }) {
-                </ListItem>
-              )}
-              {rule.mig64 && (
-                <ListItem className={classes.listItem}>
-                  <ListItemText
-                    primaryTypographyProps={{
-                      component: 'div',
-                      className: classes.primaryText,
-                    }}
-                    primary={
-                      <Fragment>
-                        64-bit Migration Opt-in
-                        {diffedProperties.includes('mig64') &&
-                          rule.scheduledChange.change_type === 'update' && (
-                            <span
-                              className={classes.propertyWithScheduledChange}
-                            />
-                          )}
-                      </Fragment>
-                    }
-                    secondary={String(rule.mig64)}
-                  />
-                </ListItem>
-              )}
-              {rule.jaws && (
-                <ListItem className={classes.listItem}>
-                  <ListItemText
-                    primaryTypographyProps={{
-                      component: 'div',
-                      className: classes.primaryText,
-                    }}
-                    primary={
-                      <Fragment>
-                        Incompatible JAWS Screen Reader
-                        {diffedProperties.includes('jaws') &&
-                          rule.scheduledChange.change_type === 'update' && (
-                            <span
-                              className={classes.propertyWithScheduledChange}
-                            />
-                          )}
-                      </Fragment>
-                    }
-                    secondary={String(rule.jaws)}
-                  />
-                </ListItem>
-              )}
-              {rule.headerArchitecture && (
-                <ListItem className={classes.listItem}>
-                  <ListItemText
-                    primaryTypographyProps={{
-                      component: 'div',
-                      className: classes.primaryText,
-                    }}
-                    primary={
-                      <Fragment>
-                        Header Architecture
-                        {diffedProperties.includes('headerArchitecture') &&
-                          rule.scheduledChange.change_type === 'update' && (
-                            <span
-                              className={classes.propertyWithScheduledChange}
-                            />
-                          )}
-                      </Fragment>
-                    }
-                    secondary={rule.headerArchitecture}
-                  />
-                </ListItem>
-              )}
-            </List>
-          </Grid>
-          <Grid item xs={12}>
-            {rule.comment && (
+        {(!rule.scheduledChange ||
+          rule.scheduledChange.change_type !== 'insert') && (
```

Comment:
The only difference in this file is that the <Grid  container /> element is now wrapped in a condition so that we don't render an empty grid container when the grid items are hidden.